jdbc/src/main/java/pl/sdacademy/java/jdbc/homework2/Homework2.java

```diff
@@ -30,7 +31,13 @@ public static void main(String[] args) {
     }
 
     public static List<Actor> getActors(SqlSessionFactory sqlSessionFactory, String query) {
-        throw new UnsupportedOperationException("TODO");
+        if (query == null || query.isBlank() || query.length() < 3) {
+            return Collections.emptyList();
+        }
+        try (SqlSession session = sqlSessionFactory.openSession()) {
+            ActorMapper actorMapper = session.getMapper(ActorMapper.class);
+            return actorMapper.getActors(query);
```

Comment:
Jak nie dasz '%' przy `LIKE`, to będzie próbował robić pełne dopasowanie 😉 dlatego nie działa.

```suggestion
            return actorMapper.getActors("%"+query+"%");
```